src/improvements/tasks/sep/008-opcm-upgrade-prestate-v300-op+ink/config.toml

```diff
@@ -4,13 +4,35 @@ templateName = "OPCMUpgradePrestateV300"
 chainId = 11155420
 name = "OP Sepolia Testnet"
 
+[[l2chains]]
+chainId = 763373
+name = "Ink Sepolia Testnet"
+
+
 [[opcmUpgrades]]
 chainId = 11155420
-absolutePrestate = "0x03ee2917da962ec266b091f4b62121dc9682bb0db534633707325339f99ee405" # TODO: change to new version with Isthmus+bfix
-expectedValidationErrors = "PDDG-ANCHORP-40,PLDG-ANCHORP-40"
+absolutePrestate = "0x03682932cec7ce0a3874b19675a6bbc923054a7b321efc7d3835187b172494b6" # op-program/v1.6.0-rc.2-64
```

Comment:
Is there a link we can point to so we can verify this prestate?

Reply:
yes working on it

Reply:
This review can be linked to from the README of the tasks: https://www.notion.so/oplabs/Isthmus-Sepolia-Mainnet-1d2f153ee162800880abe1b47910c071